app/templates/share.html

```diff
@@ -13,9 +13,32 @@ <h2><span style="color:red;">CONP Portal </span> | Share</h2>
 <!-- Content Block -->
 
 {% block content %}
-  <div>
-    <div class="jumbotron text-center">
-      <h2>Coming Soon</h2>
-    </div>
-  </div>
+<h1>Uploading data for CONP</h1>
+
+<h2>1. Metadata</h2>
+
+We use the DATS.json file format to store metadata about CONP datasets. DATS.json is a flexible machine-readable structure allowing for sophisticated representation of a wide range of different types of datasets, from neuroimaging to genomics.  A DATS.json file must be prepared for each dataset.
+
+The CONP DATS.json schema includes the following required fields:
+
+- Dataset title and description
+- Creators of the dataset
+- DOI or other unique identifier
+- License under which the dataset is distributed
+- Keywords and specification of data type
+- Version of the dataset
+- Format of primary data files in dataset
+- Total number of files in the dataset
+- Size of dataset and unit used (usually GB)
+- Web address for original dataset
+
+
+
+The schema also includes many other recommended and optional fields.  The DATS dataset schema can be found <a href="https://github.com/biocaddie/DATS/blob/master/dataset_schema.json">here</a>, and details of CONP's expansions and descriptions of what to enter in each field <a href="../CONP_DATS_fields.docx">here</a>.  The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/emmetaobrien/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template.  An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
```

Comment:
hyperlinks are identical to the text. please change color or something that would tell that it is a link.

Reply:
The error message is 'There was an error retrieving the search results.'